backends/graphics/surfacesdl/surfacesdl-graphics.cpp

```diff
@@ -1977,7 +2014,7 @@ void SurfaceSdlGraphicsManager::setMouseCursor(const void *buf, uint w, uint h,
 		assert(!_mouseOrigSurface);
 
 		// Allocate bigger surface because scalers will read past the boudaries.
-		_mouseOrigSurface = SDL_CreateRGBSurface(SDL_SWSURFACE | SDL_RLEACCEL | SDL_SRCCOLORKEY | SDL_SRCALPHA,
+		_mouseOrigSurface = SDL_CreateRGBSurface(MOUSESURFACE_FLAGS,
```

Comment:
Looking at `SDL_video.h`, it seems like `SDL_RLEACCEL`, `SDL_SRCCOLORKEY` and `SDL_SRCALPHA` are meant for internal use only, so removing them is probably the correct thing to do on all platforms.

Reply:
Done